src/store/reducers/olapStats.js

```diff
@@ -73,4 +80,10 @@ export function setOlapStatsOptions(options) {
     };
 }
 
+export function resetLoadingState() {
+    return {
+        type: RESET_LOADING_STATE,
```

Comment:
why do you use two action with exact same type? one is enough

Reply:
they are for two different reducers, types are differ by prefixes